### DISCUSSION TITLE
Attempted import error: 'en' is not exported from 'make-plural/plurals'.

### DISCUSSION THREAD

**Discussion by anotheri** [remirror/remirror#643] [General] [answered]
Hey there!

I'm Using webpack 4. And getting this error on build:

```
Creating an optimized production build...
(node:56701) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
^[Failed to compile.

./node_modules/@remirror/i18n/dist/i18n.browser.esm.js
Attempted import error: 'en' is not exported from 'make-plural/plurals'.
 @ ./node_modules/@remirror/react/dist/react.browser.esm.js
 @ ./node_modules/remirror/react/dist/remirror.browser.esm.js
 @ ./src/shared/remirrorProvider.js
```

and my `@ ./src/shared/remirrorProvider.js` file looks exactly like one of your basic examples
```
import React from 'react';
import { SocialEmojiComponent, SocialProvider } from 'remirror/react/social';

export const Remirror = () => {
  return (
    <SocialProvider>
      <SocialEmojiComponent />
    </SocialProvider>
  );
};
```

Please advice!

**Comment by anotheri** [comment]
it seems related to this file:
https://github.com/remirror/remirror/blob/next/packages/%40remirror/i18n/src/index.ts#L4-L6

**Comment by ifiokjr** [comment]
@anotheri which versions are you using?

As far as I'm aware that issue should be fixed in the `@remirror@1.0.0-next.32`.

**Reply by anotheri** [reply]
thought I had the latest one, but it looks it was `next.31`, I'll check with 32 and let you know in couple of hours.
thank you for quick answer! 🙏

**Reply by ifiokjr** [reply]
Oh, that may actually be an issue. I was thinking of something else.

I just re-read the error message and realised it's to do with `make-plurals` library. Maybe an update broke the implementation. 

Could you open an issue, I have some time to fix a few bugs over the next few hours?

**Reply by anotheri** [reply]
yeah, and just confirmed, I have all the latest packages installed but the error is still here
```
"remirror": "^1.0.0-next.32",
"@remirror/pm": "^1.0.0-next.32",
"@remirror/i18n": "^1.0.0-next.32",
```

logs on build
```
Creating an optimized production build...
(node:65563) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
Failed to compile.

./node_modules/@remirror/i18n/dist/i18n.browser.esm.js
Attempted import error: 'en' is not exported from 'make-plural/plurals'.
 @ ./node_modules/@remirror/react/dist/react.browser.esm.js
 @ ./node_modules/@remirror/react-social/dist/react-social.browser.esm.js
 @ ./node_modules/remirror/react/social/dist/remirror.browser.esm.js
 @ ./src/shared/remirrorProvider.js
```

**Reply by anotheri** [reply]
issue opened: https://github.com/remirror/remirror/issues/647

**Comment by ifiokjr** [comment, marked as answer]
Answered in #647 

> Finally! I found it! I use `file-loader` which had `.js` files excluded but had no `.mjs` excluded, so when it imported `make-plural` it went through wrong loader. Right config looks like this now:
> 
> ```
> {
>             loader: require.resolve('file-loader'),
>             // Exclude `js` files to keep "css" loader working as it injects
>             // it's runtime that would otherwise processed through "file" loader.
>             // Also exclude `html` and `json` extensions so they get processed
>             // by webpacks internal loaders.
>             exclude: [/\.js$/, /\.mjs$/, /\.html$/, /\.json$/], // <--------- added  /\.mjs$/ here
>             options: {
>               name: 'static/media/[name].[hash:8].[ext]'
>             }
>           }
> ```
> 
> Thanks for all the support, and sorry for bothering not related to your module. 🙏
